ETT-Backend/Properties/launchSettings.json

```diff
@@ -6,7 +6,9 @@
       "launchBrowser": true,
       "launchUrl": "swagger",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "API_USERNAME": "callibrity",
```

Comment:
Is this supposed to be hidden? Or is it just a default user?

Reply:
They are environment variables that we can set to be different in prod, so for local testing you can use these to send requests, but in prod we will set our username and password to be different